en/sql/tuning.rst

```diff
@@ -368,6 +385,228 @@ If you ensure that left table's row number is a lot smaller than the right table
+    #. CPU cost of a repeated routine = predicted number of leaf node keys accessed
+
+The number of pages in a non-leaf node is 1 because the index **Height**/ of the statistics is 2 and the number of pages to be read excluding the leaf node is **2 - 1**. The number of leaf node pages is obtained by multiplying **Leaf pages** by the selectivity of the predicate **index: idx term[2]** used to search the non-leaf node. Since it is **MAX(9 X 0.05, 1)**, the number of leaf node pages to be read is 1. The number of heap pages to be read is obtained by multiplying the total number of pages in the table by the selectivity of **index**/ and **filtr** conditions. Here, it is calculated as **MAX(9 X 0.05 X 0.0125,1)**, so it is calculated that 1 page must be read. Finally, the CPU cost is obtained by multiplying the selectivity of **index: idx term[2]** by the CPU weight from the total number of rows in the table. **If you calculate it as **MAX(4000 X 0.05 X 0.0025,1)**, you get 1, and if you add up all the results so far, you can see that the cost is 4. If you look at **cost: 4 card 1** in the execution plan, you can see that the cost is 4 and the expected number of rows is 1.
+
+cost of join
+-----------------------
+
+The cost of a join is generated by calculating the cost of the preceding table and the cost of the succeeding table separately, and the cost is calculated according to the join method. The following example below shows how the cost is calculated in a nested loop join method.
+
+.. code-block:: sql
+
+    create index idx1 on t2(col4);
+     
+    --;plan detail
+    select /*+ recompile */ count(*)
+    from t1 a, t2 b
+    where a.col4 = b.col4
+    and b.col1 = 1
+    and b.col3 = 1
+    and a.col2 = 2;
+
+::
+
+    Query plan:
+     
+    idx-join (inner join)
+        outer: sscan
+                   class: a node[0]
+                   sargs: term[3]
+                   cost:  19 card 1000
+        inner: iscan
+                   class: b node[1]
+                   index: idx1 term[0]
+                   sargs: term[1] AND term[2]
+                   cost:  2 card 3
+        cost:  572 card 1
+
+The preceding table is *a*/ and the succeeding table is *b*. Since the execution of the succeeding table is repeated as many times as the number of rows of the preceding table due to the nature of the nested loop join, the cost of the succeeding table can be calculated by multiplying the variable cost of the *b* table by the expected number of rows of the *a* table. **CUBRID**/ internally manages fixed costs and variable costs separately, and this information cannot be displayed in the execution plan. The variable cost of the succeeding table is approximately *0.553*, and the cost of the succeeding table incurred during the join is 553 when calculated by *0.553 * 1000*, and the final cost is *572*/ when adding the cost of the preceding table *19*/.
```

Comment:
```suggestion
The preceding table is *a* and the succeeding table is *b*. Since the execution of the succeeding table is repeated as many times as the number of rows of the preceding table due to the nature of the nested loop join, the cost of the succeeding table can be calculated by multiplying the variable cost of the *b* table by the expected number of rows of the *a* table. **CUBRID** internally manages fixed costs and variable costs separately, and this information cannot be displayed in the execution plan. The variable cost of the succeeding table is approximately *0.553*, and the cost of the succeeding table incurred during the join is 553 when calculated by *0.553 * 1000*, and the final cost is *572* when adding the cost of the preceding table *19*.
```